app/mailers/comment_mailer.rb

```diff
@@ -75,7 +75,20 @@ def comment_reply_sent_notification(comment)
     @noreply = true
     mail(
       to: @comment.comment_owner_email,
-      subject: "[#{ArchiveConfig.APP_SHORT_NAME}] Reply you left to a comment on " + (@comment.ultimate_parent.is_a?(Tag) ? "the tag " : "") + @comment.ultimate_parent.commentable_name.gsub("&gt;", ">").gsub("&lt;", "<")
+      subject: "[#{ArchiveConfig.APP_SHORT_NAME}] Reply you left to a comment on #{commentable_title(@comment)}"
     )
   end
+  
+  private
+
+  def commentable_title(comment)
```

Comment:
This is a good idea generally. However, I think we need to be able to vary the translation of the first part of the subject ("Comment on" etc) based on what the comment is left on. So each whole subject should have three total variants for the locale (tag/chapter/rest), not just the commentable tile itself.

Unfortunately, it looks like the same applies for the sentences in the emails themselves. The `commenter_pseud_or_name_*` should be fine to keep as one variable, but for the different commentables we need different full sentence variations.

I would limit the i18n to those sentences and not worry about the rest of the emails (especially the footer), since as you say it wouldn't be great to combine all that into one PR.